app/controllers/api/v1/keywords_controller.rb

```diff
@@ -6,8 +6,8 @@ class KeywordsController < ApplicationController
       before_action :authorize!
 
       def index
-        keywords = policy_scope(Keyword)
-        render json: KeywordSerializer.new(keywords).serializable_hash.to_json
+        pagination, keywords = paginated_authorized(Keyword)
+        render json: KeywordSerializer.new(keywords, meta: meta_from_pagination(pagination))
```

Comment:
```suggestion
        pagination_meta, keywords = paginated_authorized(Keyword)
        render json: KeywordSerializer.new(keywords, meta: pagination_meta)
```

Just an optional suggestion as I think we will not use the `pagination` object in other contexts for the whole project. Should we directly return the `meta` object from our custom pagination helper? 👀